splink/params.py

```diff
@@ -196,6 +200,44 @@ def _convert_params_dict_to_normalised_adjustment_data(self):
 
                 data.append(row)
         return data
+    
+    def _convert_params_dict_to_normalised_adjustment_iteration_history(self):
+        """
+        Get the data needed for a chart that shows which comparison
+        vector values have the greatest effect on match probability
+        """
+        adj_data = []
+
+        pi = gk = self.params["π"]
```

Comment:
is the ` = gk` bit a typo here?

Reply:
If it is, it's one of yours that I copied:
https://github.com/moj-analytical-services/splink/blob/master/splink/params.py#L181

Reply:
touché, haha.  i will fix, gimme a sec